stylesheets/abstracts/README.md

```diff
@@ -4,4 +4,14 @@ The `abstracts/` folder gathers all Sass tools and helpers used across the proje
 
 The rule of thumb for this folder is that it should not output a single line of CSS when compiled on its own. These are nothing but Sass helpers.
 
+## \_index.scss
+
+With the deprecation of `@import` keyword, We now use `@use` and `@forward` keywords. Unlike `@import` keyword, adding abstracts at the top of the main.scss file does not make them globally accessbile to eveything else comes afterwards, when using `@use` keyword.
+
+Because of that we need to import those abstracts using `@use` keyword each time we need them inside the individual files.
```

Comment:
```suggestion
Because of that we need to import those abstracts using `@use` keyword each time we need them inside the individual files, similar to how we would import utilities in the JavaScript world for that matter.
```

Reply:
done.